### ISSUE TITLE
Channel 6, 7, 8  function cant be assigned

### ISSUE THREAD

**Issue by chuti** [diydrones/MissionPlanner#819]
Since missionplanner 1.3.20 it is impossible to activate a function

**Comment by Flashy-GER** [comment]
I can confirm this!
since MP 1.3.20 and above (Arducopter 3.2.1), the settings cant be activated.
Please have a look on this.
I did compass calibration with MP 1.3.21, the new values are not saved.

**Comment by UAVHUB** [comment]
Confirmed here to MP 1.3.20

**Comment by meee1** [comment]
fixed here 8fdc679f8864ccf4c437a719dfacce58e043f6b7

**Comment by chuti** [comment]
Sorry, but it is not correctly fixed. It is impossible to store the values of Min and Max from channel 6 in the pixhawk.

**Comment by meee1** [comment]
please provide more info, as it works fine for me. I can go from 0 to 10000

**Comment by chuti** [comment]
If I set the values in the full parameter list it works. But if I want to set the Min and the Max value of channel6 in the Extendet Tuning it dont works right. I only can set values above siro. For example: If I set Min to 1,0000 and Max to 9,0000 it will work and in the setting of throttle accel D i can see a value between 0,001 to 0,009.
If I want to set the Min to 0,001 and the Max to 0,009 it dont works. 

i am sry for my bad English

**Comment by meee1** [comment]
make sure you are using latest mp, there was a bug on older version.
latest is 1.3.23

**Comment by chuti** [comment]
Thank you. I will check it out now. I tested 1.3.22

**Comment by Flashy-GER** [comment]
@meee1 the latest 1.3.23 is not available via Update button.
so its possible to put values with 0.xx into the mask? 
Otherwise don't have a problem to put values > 0 into it, but it should be documented.

**Comment by chuti** [comment]
I got 1.3.23 via the update button. There is no diffrent. Only integer numbers are possible. Then the values will appear as decimal numbers. Example:
Channel 6 = Throttle Accel kD
Min=1,000 (that works fine)
Min=0,001 (impossible to store)
Max=9,000 (that works fine)
Max=0,009 (impossible to store)
Throttle Accel D= a value between 0,001 and 0,009
I can live with it.

**Comment by meee1** [comment]
please try  ,  and .

**Comment by chuti** [comment]
Yes. I tried it. In my last post it is the result. But i can live with it.
